frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx

```diff
@@ -593,6 +605,24 @@ export const taxonomicFilterLogic = kea<taxonomicFilterLogicType>([
                     groupTypeIndex: type.group_type_index,
                 })),
         ],
+        eventMetadataOptions: [
+            (s) => [s.groupTypes],
+            (groupTypes) => {
+                const options = [
+                    { icon: <IconLogomark />, name: 'Distinct ID', value: 'distinct_id' },
+                    { icon: <IconLogomark />, name: 'Timestamp', value: 'timestamp' },
```

Comment:
Can you also add `event` here? Perhaps also `person_id` if it works well.

Reply:
Thinking out loud...

While I think it makes total sense to have a new event metadata `TaxonomicFilterGroupType` for this new tab/pill... I wonder if this is the right `PropertyFilterType` to add. It feels a bit restrictive. More specifically, I wonder if `EventMetadataPropertyFilter` could be merged with `HogQLPropertyFilter`.

Looking at this list of options here, we effectively have `expression operator value`, where the expression comes from a predefined list of SQL expressions. There's no backend validation... so it's effectively a generic `SQLValueFilter` or something like that.

The existing `HogQLPropertyFilter` does the `expression` part as its `key`. We could just add `operator` and `value` into it... or if making a new property filter, to make it more abstract in its name.

Reply:
> Can you also add `event` here? Perhaps also `person_id` if it works well.

Added with 94d3026689. Both work fine.

> The existing `HogQLPropertyFilter` does the `expression` part as its `key`. We could just add `operator` and `value` into it... or if making a new property filter, to make it more abstract in its name.

Doesn't that mean we'd need to apply [this logic](https://github.com/PostHog/posthog/blob/94d3026689560c309bea5b35cdba87a836cd64df/posthog/hogql/property.py#L446-L467) and [this logic](https://github.com/PostHog/posthog/blob/94d3026689560c309bea5b35cdba87a836cd64df/posthog/hogql/property.py#L160-L254) in the client?

Reply:
> Doesn't that mean we'd need to apply [this logic](https://github.com/PostHog/posthog/blob/94d3026689560c309bea5b35cdba87a836cd64df/posthog/hogql/property.py#L446-L467) and [this logic](https://github.com/PostHog/posthog/blob/94d3026689560c309bea5b35cdba87a836cd64df/posthog/hogql/property.py#L160-L254) in the client?

Hmm... well, no... it'd work exactly like the filter you now added works, just renamed to `HogQLPropertyValueFilter` or something more abstract.